charts/nx-cloud/templates/nx-cloud-message-queue-deployment.yaml

```diff
@@ -19,8 +19,8 @@ spec:
     spec:
       containers:
         - name: nx-cloud-messagequeue
-          image: nxprivatecloud/nx-cloud-messagequeue:latest
-          imagePullPolicy: Always
+          image: {{ include "nxCloud.images.messagequeue.image" . }}
```

Comment:
doesn't have to be in this PR, but should we just enable the message queue for everyone now? maybe get rid of the boolean at the top of this file?

cc. @scritchley